plugin/trino-delta-lake/pom.xml

```diff
@@ -409,6 +409,7 @@
                                 <exclude>**/TestDeltaLakeSharedGlueMetastoreWithTableRedirections.java</exclude>
                                 <exclude>**/TestDeltaLakeTableWithCustomLocationUsingGlueMetastore.java</exclude>
                                 <exclude>**/TestDeltaLakeGcsConnectorSmokeTest.java</exclude>
+                                <exclude>**/TestDeltaLakeRenameToWithGlueMetastore.java</exclude>
```

Comment:
nit: I would move this to before `TestDeltaLakeGcsConnectorSmokeTest` to make Glue tests close.